### ISSUE TITLE
deploying on Kubernetes inconsistent user

### ISSUE THREAD

**Issue by ckalas** [hasura/graphql-engine#3173]
I am running into an issue trying to deploy on AKS, until now I've been running the exact same setup but in docker (connecting to an azure Postgres instance). The last line of the log makes me think it is somehow parsing the Postgres user incorrectly - it says `auth failed for user "\NAME\"`, but in the earlier log messages it is correctly reporting the user is "NAME@az-postgres". As per the docs, I specified my connection in the deployment file as `postgres://NAME%40az-postgres:password@az-postgres.postgres.database.azure.com:5432/postgres`. 

Given that the connection string works running in docker I am a bit confused as to where the problem is coming from.

`{"timestamp":"2019-10-18T08:05:01.235+0000","level":"info","type":"startup","detail":{"kind":"server_configuration","info":{"live_query_options":{"fallback_options":{"refetch_delay":1000},"multiplexed_options":{"batch_size":100,"refetch_delay":1000}},"transaction_isolation":"ISOLATION LEVEL READ COMMITTED","enabled_log_types":["unstructured","jwk-refresh-log","ws-server","http-log","schema-sync-thread","websocket-log","startup","webhook-log","pg-client","telemetry-log","metadata","event-trigger"],"server_host":"HostAny","enable_allowlist":false,"log_level":"info","auth_hook_mode":null,"use_prepared_statements":true,"unauth_role":null,"stringify_numeric_types":false,"enabled_apis":["metadata","graphql","config","pgdump"],"enable_telemetry":true,"enable_console":true,"auth_hook":null,"jwt_secret":null,"cors_config":{"allowed_origins":"*","disabled":false,"ws_read_cookie":null},"console_assets_dir":null,"admin_secret_set":true,"port":8080}}}
{"timestamp":"2019-10-18T08:05:01.235+0000","level":"info","type":"startup","detail":{"kind":"postgres_connection","info":{"database":"postgres","retries":1,"user":"NAME@az-postgres","host":"pg-bulkpredict.postgres.database.azure.com","port":5432}}}
{"timestamp":"2019-10-18T08:05:01.235+0000","level":"info","type":"startup","detail":{"kind":"db_init","info":"the state is already initialised"}}
{"timestamp":"2019-10-18T08:05:01.235+0000","level":"info","type":"startup","detail":{"kind":"db_migrate","info":"already at the latest version. current version: \"22\""}}
{"timestamp":"2019-10-18T08:05:01.235+0000","level":"info","type":"startup","detail":{"kind":"schema-sync","info":{"thread_id":"ThreadId 18","instance_id":"9759a0e5-1412-41a9-9b2d-649c08f437cd","message":"listener thread started"}}}
{"timestamp":"2019-10-18T08:05:01.235+0000","level":"info","type":"startup","detail":{"kind":"schema-sync","info":{"thread_id":"ThreadId 19","instance_id":"9759a0e5-1412-41a9-9b2d-649c08f437cd","message":"processor thread started"}}}
{"timestamp":"2019-10-18T08:05:01.235+0000","level":"info","type":"startup","detail":{"kind":"event_triggers","info":"preparing data"}}
{"internal":"FATAL:  authentication failed for user \"NAME": too many failed login attempts\nFATAL:  authentication failed for user \"NAME\": too many failed login attempts\n","path":"$","error":"connection error","code":"postgres-error"}`

**Comment by shahidhk** [comment]
@ckalas I hope got this issue resolved?

**Comment by Amsh-Jiva** [comment]
Hi, We are facing similar issue. Can you pls share the resolution/fix?

**Comment by ismaelvacco** [comment]
yeah. I solve this issue replacing the at sign symbol with the percentage version. For example from `username@server-id` to `username%40server-id`

I hope have help you!
